internal/stack/stacks.go

```diff
@@ -136,19 +148,43 @@ func (p *stackParser) parseStack(line string) (Stack, error) {
 		fullStack.WriteString(line)
 		fullStack.WriteByte('\n') // scanner trims the newline
 
-		// The first line after the header is the top of the stack.
+		if len(line) == 0 {
```

Comment:
when does this happen?

Reply:
There's an empty line between each goroutine's trace. I don't want to hard code that, though, and I want to allow the format to allow for an empty line in the trace (since the format isn't fixed), so it made sense to skip it.